sdk/examples/how_tos/account_wallet/transaction.rs

```diff
@@ -54,7 +54,9 @@ async fn main() -> Result<()> {
     // Find first output unlockable by the account address
     let input = *account
         .client()
-        .basic_output_ids([QueryParameter::Address(account_address)])
```

Comment:
Just making sure, does this example actually need only_address_unlock_condition ? Or should it be UnlockableByAddress?

Reply:
only_address_unlock_condition() is the safer option, the other one could fail